sympy/core/function.py

```diff
@@ -922,8 +929,10 @@ class Derivative(Expr):
     automatically simplified in a fairly conservative fashion unless the
     keyword ``simplify`` is set to False.
 
-        >>> from sympy import sqrt, diff
-        >>> from sympy.abc import x
+        >>> from sympy import cos, sin, sqrt, diff, Function
+        >>> from sympy.abc import x, y, z
+        >>> f, g = map(Function, 'fg')
```

Comment:
You should use `symbols(cls=Function)` in docstrings.

Reply:
done